src/odinapi/views/level2.py

```diff
@@ -1012,9 +1032,21 @@ def _fetch(self, version, project):
         except ValueError as err:
             raise BadRequest(str(err))
         db = level2db.Level2DB(project)
-        # TODO: Limit/paging
+        products = param['products']
         meas_iter = db.get_measurements(param.pop('products'), **param)
-        return meas_iter
+        if version == 'v4':
+            return meas_iter
+        scans, next_min_scanid = level2db.get_valid_collapsed_products(
+            list(meas_iter), param['document_limit'])
+        url = url_for(
+            'level2viewlocations', project=project, version=version,
+            _external=True)
+        headers = {}
+        if next_min_scanid:
```

Comment:
@skymandr indicated that `0` is actually a possible scanid, if so shouldn't it be `if next_min_scanid is not None`?